### PR TITLE
Fixed packaging and publishing NuGet packages

### PR DESCRIPTION
**Pull request by xperiandri**
1. Fixed `The 'NoBuild' property was set to true but the 'Build' target was invoked. ` error for the `Relay` project
2. Fixed ability to skip already published packages and continue with not yet published using `--skip-duplicate` parameter

### CODE REVIEW

File: build/DotNet.fs [hunk 1]
```diff
@@ -27,7 +27,9 @@ module DotNet =
         if Environment.isUnix then
             Environment.environVar "HOME" @@ ".dotnet"
         else
-            Environment.environVar "LocalAppData" @@ "Microsoft" @@ "dotnet"
+            Environment.environVar "LocalAppData"
+            @@ "Microsoft"
+            @@ "dotnet"
```
Comment (mickhansen) [inline build/DotNet.fs:32]: Did any logic in this file change, or is it only formatting changes?

**Review by mickhansen — APPROVED** [review]
I'm not too well versed in how packaging works, but overall LGTM.

**Review by GBirkel — APPROVED** [review]

**Review by valbers — APPROVED** [review]
Fine by me

File: build/DotNet.fs (hunk 1 above)
Comment (xperiandri) [reply build/DotNet.fs:32]: Only formatting.

Only this added
https://github.com/fsprojects/FSharp.Data.GraphQL/blob/76af489fe2ef701754827d34e38a02bab56021e2/build/NuGet.fs#L576-L578

**Pull request merged**